linkerd/core/src/main/scala/io/buoyant/linkerd/DefaultInterpreterInitializer.scala

```diff
@@ -48,7 +50,14 @@ case class ConfiguredNamersInterpreter(namers: Seq[(Path, Namer)])
       // For now, punt to the default interpreter. Later, we
       // should consider removing the finagle Namer.global, which
       // provides /$/ names.
-      case Nil => Namer.global.lookup(path)
+      case Nil =>
+        Namer.global.lookup(path).transform {
```

Comment:
yah, I wanted to use `.handle{}`, but it expects you to like... actually handle the error, not turn it into another better error